src/derived/root.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+import React from 'react'
+import Grid from '../grid'
+
+export default function Root(props: { [string]: any }) {
```

Comment:
Only a subset since you shouldn't set margin or padding. It can def be better than this definition though 😆